src/fosslight_binary/_jar_analysis.py

```diff
@@ -196,4 +196,5 @@ def analyze_jar_file(path_to_find_bin):
         logger.debug(f"Error to get depency Info in jar_contets: {ex}")
         success = False
 
+    os.remove(json_file)
```

Comment:
Wrap it in a try except statement and delete it after checking if the file exists.

Reply:
Added it. Thx